File: URIOJ/uri2663.cpp
/*
 * Leonardo Deliyannis Constantin
 * URI 2663 - Fase (Maratona SBC 2017)
 */

#include <stdio.h>
#include <algorithm>
using namespace std;
#define MAX 1123

bool revcmp(const int &a, const int &b) {
	return a > b;
}

int main(){
	int N, K, v[MAX], i;
	int ans;
	while(scanf("%d", &N) != EOF){
		scanf("%d", &K);
		for(i = 0; i < N; i++){
			scanf("%d", &v[i]);
		}
		sort(v, v+N, revcmp);
		ans = K;
		while(v[ans-1] == v[ans]) ans++;
		printf("%d\n", ans);
	}
	return 0;
}

File: Oficinas2018/Mai03/uri2663.py
# -*- coding: utf-8 -*-
# Leonardo Deliyannis Constantin
# URI 2663 - Fase

N = int(input())
K = int(input())
v = []
for i in range(0, N):
    x = int(input())
    v.append(x)
v.sort(reverse = True)

while K < N and v[K-1] == v[K]:
    K += 1

print(K)
